packages/ie11-custom-properties/README.md

```diff
@@ -0,0 +1,13 @@
+# @fluentui/ie11-custom-properties
+
+**Ie11 Polyfill for [Fluent UI React](https://developer.microsoft.com/en-us/fluentui)**
+
+Package to hold ie11 polyfill
```

Comment:
Would be good to specify what this is a polyfill for. There are lots of other types of polyfills besides for CSS variables (like for adding Promise, newer array methods, etc).

Reply:
Added more info to the README